src/Action.hs

```diff
@@ -104,12 +110,6 @@ instance PersistM r => ActionPersist r (Action r) where
 instance MonadIO (Action r) where
     liftIO = Action . liftIO
 
-instance PersistM r => PersistM (Action r) where
```

Comment:
If we are convinced you know what you are doing (as for me, I'm only guessing based on the comments for the `persistent` vel `atomically` operation), please just revert the commit:  "Remove the dangerous PersistM instance for Action. When using the instance there is little control over authorization, atomicity, etc. This is best handled explicitly."